src/digraph6.rs

```diff
@@ -0,0 +1,122 @@
+use crate::{get_edge_iter_with_weights, StablePyGraph};
+use crate::graph6::{utils, IOError, GraphConversion};
+use pyo3::prelude::*;
+use pyo3::types::PyAny;
+use petgraph::graph::NodeIndex;
+use petgraph::algo;
+
+/// Directed graph implementation (extracted from graph6.rs)
+#[derive(Debug)]
+pub struct DiGraph {
```

Comment:
Same comment from `Graph`, rename it to something better. We made this naming mistake with GraphML, but we can do better on newer code